### PR TITLE
Add A-A failover scenario test

### PR DESCRIPTION
**Pull request by uglide**
Basic scenario test for AA failover.

### CODE REVIEW

**Comment by uglide** [comment]
run scenario tests

**Comment by uglide** [comment]
| Testcase                                                |   Errors |   Failures |   Skipped |   Total |
|---------------------------------------------------------|----------|------------|-----------|---------|
| redis.clients.jedis.scenario.ConnectionInterruptionTest |        0 |          0 |         0 |       4 |
| redis.clients.jedis.scenario.ClusterTopologyRefreshTest |        0 |          0 |         0 |       1 |
| redis.clients.jedis.scenario.ActiveActiveFailoverTest   |        0 |          0 |         1 |       1 |


---- [Details for mantainers](https://github.com/redis-developer/cae-client-testing/actions/runs/10529303939)

**Comment by uglide** [comment]
| Testcase                                                |   Errors |   Failures |   Skipped |   Total |
|---------------------------------------------------------|----------|------------|-----------|---------|
| redis.clients.jedis.scenario.ConnectionInterruptionTest |        0 |          0 |         0 |       4 |
| redis.clients.jedis.scenario.ClusterTopologyRefreshTest |        0 |          0 |         0 |       1 |
| redis.clients.jedis.scenario.ActiveActiveFailoverTest   |        0 |          0 |         0 |       1 |


---- [Details for mantainers](https://github.com/redis-developer/cae-client-testing/actions/runs/10529515334)

File: src/test/java/redis/clients/jedis/scenario/ActiveActiveFailoverTest.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+      public String getCurrentClusterName() {
+        return currentClusterName;
+      }
+
+      @Override
+      public void accept(String clusterName) {
+        this.currentClusterName = clusterName;
+        log.info(
+            "\n\n====FailoverEvent=== \nJedis failover to cluster: {}\n====FailoverEvent===\n\n",
+            clusterName);
```
[the remark this answers is not in the document]
Comment (uglide) [reply src/test/java/redis/clients/jedis/scenario/ActiveActiveFailoverTest.java:82]: It's a good suggestion, but we run scenario tests separately, so it's not yet a problem there. Still, we should look out for this problem.

[the remark this answers is not in the document]
Comment (sazzad16) [reply src/test/java/redis/clients/jedis/scenario/ActiveActiveFailoverTest.java:82]: Ah, right! I missed that part and thought at least something is going to run in Jedis CI.

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**